src/mam4xx/mo_gas_phase_chemdr.hpp

```diff
@@ -82,11 +106,15 @@ void perform_atmospheric_chemistry_and_microphysics(
     const View1D &linoz_PmL_clim_icol, const View1D &linoz_dPmL_dO3_icol,
     const View1D &linoz_dPmL_dT_icol, const View1D &linoz_dPmL_dO3col_icol,
     const View1D &linoz_cariolle_pscs_icol, const Real eccf,
-    const Real adv_mass_kg_per_moles[gas_pcnst],
+    const Real adv_mass_kg_per_moles[gas_pcnst], const Real mmr[gas_pcnst],
+    const Real fraction_landuse[mam4::mo_drydep::n_land_type],
+    const int col_index_season[mam4::mo_drydep::n_land_type],
```

Comment:
@overfelt For col_index_season, please see PR3108. This PR was merged into the master branch in the scream repo. However, I do not see these changes in the master branch of E3SM. We may need to wait or request for these branches to be synced.